core/domain/question_domain.py

```diff
@@ -240,6 +240,31 @@ def _convert_state_v30_dict_to_v31_dict(cls, question_state_dict):
                 audio_metadata['duration_secs'] = 0.0
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v31_dict_to_v32_dict(cls, question_state_dict):
+        """Converts from version 31 to 32. Version 32 adds a new
+        customization arg to SetInput interaction which allows
+        creators to add custom text to the "Add" button.
+
+        Args:
+            question_state_dict: dict. A dict where each key-value pair
+                represents respectively, a state name and a dict used to
+                initialize a State domain object.
+
+        Returns:
+            dict. The converted question_state_dict.
+        """
+        if question_state_dict['interaction']['id'] == 'SetInput':
+            customization_args = question_state_dict[
+                'interaction']['customization_args']
+            customization_args.update({
+                'buttonText': {
+                    'value': 'Add item'
+                }
+            })
+
+        return question_state_dict
+
```

Comment:
There's another PR (PR #8718 ) by @kevintab95 that also updates state from v31 to v32. Also Kevin's PR adds a question_id param, you two may want to coordinate

Reply:
Yep, but I have some comments to address before it can go in. Let's get this merged first and then I'll fix the merge conflicts in addition to addressing the comments. Thanks!